processors/base/writers/location.py

```diff
@@ -34,7 +40,7 @@ def write_location(conn, location, source_id, trial_id=None):
     timestamp = datetime.datetime.utcnow()
 
     # Get name
-    name = helpers.clean_string(location['name'])
+    name = helpers.clean_string(extract_CanonicalName(location['name']))
```

Comment:
I understand your points @vitorbaptista and are good ones.
I didn't treat the case of having different ranges of similarity because 80% is a high number and I didn't saw any countries that gonna have so similar names. (80% means small number of letters like "of" or "s", but if you think can be a good improvement I can add that)
And I the fuzz.ratio do the lowercasing process.